Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3542,17 +3479,17 @@ def panorama_edit_edl(edl_name, element_to_change, element_value):
     return result, edl_output
 
 
-def panorama_edit_edl_command():
+def panorama_edit_edl_command(args: dict):
     """
     Edit an EDL
     """
-    edl_name = demisto.args()['name']
-    element_to_change = demisto.args()['element_to_change']
-    element_value = demisto.args()['element_value']
+    edl_name = args['name']
+    element_to_change = args['element_to_change']
+    element_value = args['element_value']
 
     result, edl_output = panorama_edit_edl(edl_name, element_to_change, element_value)
 
-    demisto.results({
+    return_results({
```

Comment:
are you sure this is legit?

Reply:
yeah, also required (warning level) by sdk lint

Reply:
you are correct, seen it just now.